packages/flame_test/lib/src/flame_test.dart

```diff
@@ -116,10 +120,26 @@ class GameTester<T extends Game> {
   ///
   /// Use [verify] closure to make verifications/assertions.
   @isTest
+  @Deprecated('Use testGameWidget instead')
   void widgetTest(
     String description,
     WidgetVerifyFunction<T>? verify,
   ) {
+    testGameWidget(description, verify: verify);
+  }
+
+  /// Creates a [Game] specific test case with given [description]
+  /// which runs inside the Flutter test environment.
+  ///
+  /// Use [setUp] closure to prepare your game instance (e.g. add components to
+  /// it)
+  /// Use [verify] closure to make verifications/assertions.
+  @isTest
+  void testGameWidget(
```

Comment:
Not really, just tested and I can still filter a single test:

```
flutter test --name "renders correctly"
00:04 +1: All tests passed!
```

Reply:
<img width="506" alt="Screen Shot 2022-03-30 at 10 02 37 AM" src="https://user-images.githubusercontent.com/4231472/160891297-e37e53ce-1dcf-4980-ad59-ceead0aff1e8.png">

Reply:
![Screenshot 2022-03-30 at 15 01 50](https://user-images.githubusercontent.com/835641/160901294-253992c8-d1ea-4325-9761-8a622ae90463.png)

Probably an issue with your IDE/Plugin/etc

Reply:
That's Android Studio, one of the two [most popular IDEs](https://blog.logrocket.com/best-ides-flutter-2022/) for Flutter. I don't have exact statistics, but it probably has around 50% market share.

Reply:
> That's Android Studio, one of the two [most popular IDEs](https://blog.logrocket.com/best-ides-flutter-2022/) for Flutter. I don't have exact statistics, but it probably has around 50% market share.

It is still a problem with the Android Studio Flutter plugin in then, because this should be expected behavior

Reply:
The description of the `@isTest` annotation says:
> /// Used to annotate a test framework function that runs a single test.
///
/// Tools, such as IDEs, can show invocations of such function in a file
/// structure view to help the user navigating in large test files.
///
/// The first parameter of the function must be the description of the test.

There is no mention of this being applicable to methods; so it is even not clear that Android Studio's behavior qualifies as a bug.

But regardless of whether it's a bug or a missing feature, it is unlikely to be fixed soon. In the meanwhile, close to half of all Flame users will be affected by this problem.

Reply:
> close to half of all Flame users will be affected by this problem.

I doubt that half of all Flame users run our tests, I wish it was true though. :wink:

Reply:
`flame_test` is a public library though; so any user who writes tests for their own game can be affected. I concede though that it is probably less common than we'd like.

Still, this is a separate problem, one that we can address at the documentation level. For example, if all our tutorials included discussions on how to add tests, or if we had a separate tutorial dedicated just to writing tests, then there would be much more uptake.